src/custom/state/price/hooks.ts

```diff
@@ -59,6 +59,12 @@ export const useClearQuote = (): ClearPriceCallback => {
   return useCallback((clearQuoteParams: ClearQuoteParams) => dispatch(clearQuote(clearQuoteParams)), [dispatch])
 }
 
-export const useAllQuoteDispatch = (): [SetLoadPriceCallback, AddPriceCallback, ClearPriceCallback] => {
-  return [useLoadingQuote(), useUpdateQuote(), useClearQuote()]
+interface QuoteDispatchers {
```

Comment:
nitpick. Functionally I don't understand why we need to expose the word `dispatcher`. This is implementation details since it's redux terminology. I don't have a good alternative, but sth like `useQuoteUpdater` since it gives some functions to update the quote state.

Reply:
i mean for me it's clear. throwback to the pure redux days when this is how we named things i guess. i can change it to whatever you want and/or suggest a name i can commit it now

Reply:
its nitpick, u can ignore. I just don't like to expose this underlying tech. 
I would even understand more to call them actions, like `useQuoteActions` which although is redux terminology it's also a good functional name